README.md

```diff
@@ -7,10 +7,10 @@
 **OK.css** is a
 [classless CSS framework](https://css-tricks.com/no-class-css-frameworks/). Dropping it into your HTML will make your page look decent — no need to reference documentation, think about responsiveness, or account for browser differences as long as your markup is semantically-correct.
 
-To use it, you can [download the CSS file directly](https://okcss.netlify.app/ok.min.css) or add the following line to your HTML `<head>`:
+To use it, you can [download the CSS file directly](https://cdn.jsdelivr.net/gh/andrewh0/okcss@1/dist/ok.min.css) or add the following line to your HTML `<head>`:
```

Comment:
I think all the `@1`s in here should be `@v1.0.1` as of Git HEAD instead, matching Git tag names. Am I missing something?

Reply:
Ah, yes thank you! Will include a `v1` here instead of a specific version to take advantage of jsdelivr's ability to target version ranges.

```suggestion
To use it, you can [download the CSS file directly](https://cdn.jsdelivr.net/gh/andrewh0/okcss@v1/dist/ok.min.css) or add the following line to your HTML `<head>`:
```

Reply:
You mean you are going to use semver and `v1` will resolve to say `v1.20.300` if that's the latest semver match?

Reply:
Correct!